src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Win32.cs

```diff
@@ -738,115 +737,12 @@ private static bool TryCompareTimeZoneInformationToRegistry(in TIME_ZONE_INFORMA
-                    filePath, language, ref languageLength,
-                    fileMuiPath, ref fileMuiPathLength, ref enumerator);
-                fileMuiPath[Interop.Kernel32.MAX_PATH] = '\0';
-
-                if (!succeeded)
-                {
-                    // Recurse to search using the parent of the desired culture.
-                    if (cultureInfo.Parent.Name != string.Empty)
-                    {
-                        return GetFileMuiPath(filePath, cultureInfo.Parent);
-                    }
-
-                    // Final fallback, using the preferred installed UI language.
-                    enumerator = 0;
-                    language[0] = '\0';
-                    succeeded = Interop.Kernel32.GetFileMUIPath(
-                        Interop.Kernel32.MUI_USER_PREFERRED_UI_LANGUAGES,
-                        filePath, language, ref languageLength,
-                        fileMuiPath, ref fileMuiPathLength, ref enumerator);
-                    fileMuiPath[Interop.Kernel32.MAX_PATH] = '\0';
-
-                    if (succeeded)
-                    {
-                        return new string(fileMuiPath);
-                    }
-
-                    return string.Empty;
-                }
-
-                // Lookup succeeded.  Check for exact match to the desired culture.
-                language[Interop.Kernel32.LOCALE_NAME_MAX_LENGTH] = '\0';
-                ReadOnlySpan<char> lang = MemoryMarshal.CreateReadOnlySpanFromNullTerminated(language);
-                if (lang.Equals(cultureInfo.Name, StringComparison.OrdinalIgnoreCase))
-                {
-                    return new string(fileMuiPath);
-                }
-
-                // Check for match of any parent of the language returned to the desired culture.
-                var ci = CultureInfo.GetCultureInfo(lang.ToString());
-                while (ci.Parent.Name != string.Empty)
-                {
-                    if (ci.Parent.Name.Equals(cultureInfo.Name, StringComparison.OrdinalIgnoreCase))
-                    {
-                        return new string(fileMuiPath);
-                    }
-
-                    ci = ci.Parent;
-                }
-
-                // Not found yet.  Continue with next iteration.
-            }
-        }
 
         /// <summary>
-        /// Helper function for retrieving a localized string resource via MUI.
-        /// The function expects a string in the form: "@resource.dll, -123"
-        ///
-        /// "resource.dll" is a language-neutral portable executable (LNPE) file in
-        /// the %windir%\system32 directory.  The OS is queried to find the best-fit
-        /// localized resource file for this LNPE (ex: %windir%\system32\en-us\resource.dll.mui).
-        /// If a localized resource file exists, we LoadString resource ID "123" and
-        /// return it to our caller.
+        /// Try to find the time zone resources Dll matching the CurrentUICulture or on eof its parent cultures.
```

Comment:
> on eof its parent cultures

?  Is that supposed to say "on any of"?

Reply:
it supposes to say `or one of its parent cultures` :-)